src/firefly/java/edu/caltech/ipac/firefly/server/query/EmbeddedDbProcessor.java

```diff
@@ -151,7 +151,6 @@ public DataGroupPart getData(ServerRequest request) throws DataAccessException {
         } finally {
             lockChecker.unlock();
         }
-
         // make sure multiple requests for the same data waits for the first one to create before accessing.
```

Comment:
It's not a big deal.  But, in general, you should clean up your touched file so that it does not create fake modification history.